spec/tapioca/compilers/dsl/active_resource_spec.rb

```diff
@@ -0,0 +1,207 @@
+# typed: false
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe("Tapioca::Compilers::Dsl::ActiveResource") do
+  before(:each) do
+    require "tapioca/compilers/dsl/active_resource"
+  end
+
+  subject do
+    Tapioca::Compilers::Dsl::ActiveResource.new
+  end
+
+  describe("#initialize") do
+    def constants_from(content)
+      with_content(content) do
+        subject.processable_constants.map(&:to_s).sort
+      end
+    end
+
+    it("gathers no constants if there are no ActiveResource classes") do
+      assert_empty(subject.processable_constants)
+    end
+
+    it("gathers only ActiveResource constants ") do
+      content = <<~RUBY
+        class Post < ActiveResource::Base
+        end
+
+        class Product < ActiveRecord::Base
+        end
+
+        class User
+        end
+      RUBY
+
+      assert_equal(constants_from(content), ["Post"])
```

Comment:
With the above suggestion:
```suggestion
      assert_equal(constants_from(content), ["Post", "Product"])
```